pkg/gadget-collection/gadgets/snapshot/process/gadget.go

```diff
@@ -91,7 +93,9 @@ func (t *Trace) Collect(trace *gadgetv1alpha1.Trace) {
 		MountnsMap:  mountNsMap,
 		ShowThreads: showThreads,
 	}
-	events, err := tracer.RunCollector(config, t.helpers)
+	localLog := log.New()
+	localLog.SetLevel(log.GetLevel())
```

Comment:
`log.StandardLogger()` can be used instead to avoid creating a new log instance.